### ISSUE TITLE
Improve readability: Git package dependencies on separate lines

### ISSUE THREAD

**Issue by artur-sannikov** [b-rodrigues/rix#158]
Now, if a package from GitHub has many dependencies, it results in a very long line after `propagatedBuildInputs = builtins.attrValues`. It's possible in `nix` to have each of these packages on a separate line, which significantly improves code readability.

For example,

```nix
    propagatedBuildInputs = builtins.attrValues {
    inherit (pkgs.rPackages)
      SummarizedExperiment
      SingleCellExperiment
      TreeSummarizedExperiment
      MultiAssayExperiment
      MASS
      ape
      decontam
      vegan
      BiocGenerics
      S4Vectors
      IRanges
      Biostrings
      DECIPHER
      BiocParallel
      DelayedArray
      DelayedMatrixStats
      scuttle
      scater
      DirichletMultinomial
      rlang
      dplyr
      tibble
      tidyr
      bluster
      MatrixGenerics;
    };
```

**Comment by b-rodrigues** [comment]
partially fixed, but buildInputs of github packages need the same treatment

**Comment by philipp-baumann** [comment]
#174

**Comment by philipp-baumann** [comment]
I addressed this in #195 thanks for the input @artur-sannikov
